orchestrator/broadcaster.go

```diff
@@ -3,12 +3,29 @@ package orchestrator
 import (
 	"context"
 
+	"github.com/celestiaorg/orchestrator-relayer/p2p"
+
 	"github.com/celestiaorg/orchestrator-relayer/types"
 )
 
-type BroadcasterI interface {
-	// BroadcastConfirm broadcasts an attestation confirm to the P2P network.
-	BroadcastConfirm(ctx context.Context, confirm types.AttestationConfirm) (string, error)
+type Broadcaster struct {
+	QgbDHT *p2p.QgbDHT
+}
+
+func NewBroadcaster(qgbDHT *p2p.QgbDHT) *Broadcaster {
+	return &Broadcaster{QgbDHT: qgbDHT}
 }
 
-// Note: broadcaster implementation will be done after defining the P2P interfaces.
+func (b Broadcaster) BroadcastDataCommitmentConfirm(ctx context.Context, nonce uint64, confirm types.DataCommitmentConfirm) error {
```

Comment:
minor optional nit on naming, perhaps "provide", or "put" would be better here

Reply:
[76e7743](https://github.com/celestiaorg/orchestrator-relayer/pull/115/commits/76e77432321b3f7cd34409673e7fd42a7db2b8c0)